src/Tabs/Tabs.tsx

```diff
@@ -17,11 +17,13 @@ import {
   TabButton,
   PlusWrapper,
 } from "./Tabs.styled"
+import { Spinner } from ".."
```

Comment:
No cyclic dependencies please.

```suggestion
import { Spinner } from "../Spinner/Spinner"
```